actions/kexec/v1/Dockerfile

```diff
@@ -1,16 +1,26 @@
 # syntax=docker/dockerfile:experimental
 
 # Build kexec
-FROM golang:1.15-alpine as kexec
+FROM golang:1.14-alpine as kexec
```

Comment:
I'm curious about this downgrade.

Reply:
This was recommended by the U-Root people.

Reply:
Works for me. Thanks!